src/overlays/actors/ovl_En_Syateki_Man/z_en_syateki_man.c

```diff
@@ -1349,7 +1349,7 @@ void EnSyatekiMan_Town_RunGame(EnSyatekiMan* this, PlayState* play) {
             func_801A2C20();
             this->actionFunc = EnSyatekiMan_Town_EndGame;
             if (this->score == 50) {
-                func_801A3098(NA_BGM_GET_ITEM | 0x900);
+                Audio_PlayFanfare(NA_BGM_GET_ITEM | 0x900);
```

Comment:
It's complicated, but 0x800 and 0x900 specifically seems serve no function purpose in terms of actually playing the sequence themselves. All it does is get attached to the seqId, so that if you check for the active seqId playing, you need to make sure to have the same `0x800` or `0x900` flag attached to it. So in that sense, it should not be broken up.

The file dealing with these flags hasn't been PR'd yet, but I can also point you to that code if you're interested in learning more